io.github.xaos_project.XaoS.yml

```diff
@@ -0,0 +1,41 @@
+app-id: io.github.xaos_project.XaoS
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: xaos
+rename-icon: xaos
+rename-desktop-file: xaos.desktop
+rename-appdata-file: xaos.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+
+modules:
+  - name: xaos
+    buildsystem: qmake
+    no-make-install: true
+    post-install:
+      - install -Dm 755 bin/xaos ${FLATPAK_DEST}/bin/xaos
+      - mkdir -p ${FLATPAK_DEST}/share/XaoS/
+      - cp -a examples tutorial catalogs ${FLATPAK_DEST}/share/XaoS/
+      - install -Dm 644 xdg/xaos.png ${FLATPAK_DEST}/share/icons/hicolor/256x256/apps/xaos.png
+      - install -Dm 644 xdg/*.desktop ${FLATPAK_DEST}/share/applications/xaos.desktop
+      - install -Dm 644 xdg/xaos.appdata.xml ${FLATPAK_DEST}/share/metainfo/xaos.appdata.xml
+    sources:
+      - type: git
+        url: https://github.com/xaos-project/XaoS.git
+        tag: release-4.2.1
+        commit: 632a48b088326a916e7c6f8fdb4a549ddfcbc7c0
+        x-checker-data:
+          type: git
+          tag-pattern: ^release-([\d.]+)$
+      - type: shell
+        commands:
+          - sed -i 's@#define DATAPATH "/usr/share/XaoS"@#define DATAPATH "/app/share/XaoS"@' src/include/config.h
+      - type: patch
+        path: appdata1.patch
+      - type: patch
+        path: appdata2.patch
+      - type: patch
+        path: appdata3.patch
```

Comment:
not sure why you have 3 patches like that.

Reply:
To track upstream changes separately to my pull request.